components/dictionary.txt

```diff
@@ -0,0 +1,98 @@
+ableist
+al
+Alevin
+ALSF
+ArrayExpress
+AUCell
+Benchmarking
+bioinformatics
+bp
+Bruning
+bulletpoint
+Bundler
+Buttner
+CanFam
+Canis
+Cao
+Carpentries
+ccdatalab
+chatbot
+cheatsheet
+cheatsheets
+checkmarks
+compositional
+customizable
+Danio
+DCP
+DESeq
+DESeqDataset
+discoverable
+dropdown
+DropSeq
+ecure
+ecursively
+EmptyDrops
+Ensembl
+epitope
+et
+familiaris
+FileZilla
+FileZilla's
+Gb
+GDebi
+GRCh
+GRCm
+GRCz
+GSEA
+Haghverdi
+HCA
+Hemburg
+hg
+Hippen
+Homebrew
+jekyll
+Korsunsky
+Luecken
+Lun
+LTS
+macOS
+miQC
+modifided
+Muris
+musculus
+nonconsensual
+OSCA
+PHI
+PII
+pre
+README
+rerio
+rstudio
```

Comment:
Do we want the uncapitalized version?

Reply:
Went back and looked at how this got in there. Fixed! https://github.com/AlexsLemonade/training-specific-template/pull/131/commits/40088beaee45fd8929ffc7006cc51ecf36cc7e5f